images/echoserver/echoserver.go

```diff
@@ -33,6 +37,14 @@ type RequestAssertions struct {
 	Headers map[string][]string `json:"headers"`
 
 	Context `json:",inline"`
+
+	TLSAssertions *TLSAssertions `json:"tls,omitempty"`
```

Comment:
Let's call this field "TLS" to match the json tag.

Reply:
Fixed.